tools/ppa-upload.sh

```diff
@@ -97,7 +103,7 @@ series = ubuntu.getSeries(name_or_version=os.environ['RELEASE'])
 
 mir_team = lp.people["mir-team"]
 rc_ppa = mir_team.getPPAByName(name="wlcs-rc")
-release_ppa = mir_team.getPPAByName(name="wlcs")
+release_ppa = mir_team.getPPAByName(name="release")
```

Comment:
Hm. This is going to cause WLCS releases to immediately hit the Mir release PPA, which was not our plan. The plan was to manually copy from a WLCS release PPA (which it seems I neglected to actually set up!) into Mir release PPA when we were ready, as new WLCS releases can result in Mir CI failing.

That said, new WLCS releases aren't exactly a daily occurrence *and* we now have very good tools for marking WLCS tests as expected-fail, so maybe direct-to-Mir-release *is* the right choice now.

Reply:
Yeah I believe we need to re-think this a bit. Do we need separate `wlcs-` PPAs? Don't we hide things by not running Mir `-dev` with WLCS `dev` until such a time they meet in the Mir release PPA?

Reply:
The idea was that if we push WLCS builds straight into Mir CI then WLCS can break Mir CI without any code changes in Mir, which is a bit annoying.

That said, it would be easy to notice breakage and, as I said, we now have good tools for `XFAIL`ing tests, so maybe we *should* be breaking Mir CI early?

Reply:
Which is to say: @AlanGriffiths, @wmww, @graysonguarino: opinions?

Reply:
I'd 👍 just using `mir-team/dev`, `/rc`, `/release` PPAs for WLCS.

Reply:
I don't think there's an approach without drawbacks. Let's just try it, if it breaks we will fix.